dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -91,8 +103,9 @@ public OrganizationResource(SessionFactory sessionFactory) {
     @Path("{organizationId}/approve/")
     @ApiOperation(value = "Approves an organization.", authorizations = {
         @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, notes = "Admin/curator only", response = Organization.class)
-    public Organization approveOrganization(@ApiParam(hidden = true) @Auth User user,
-        @ApiParam(value = "Organization ID.", required = true) @PathParam("organizationId") Long id) {
+    @Operation(operationId = "approveOrganization", summary = "Approves an organization.", description = "Approves the organization with the given id. Admin/curator only.", security = @SecurityRequirement(name = "bearer"))
```

Comment:
Was like this already, but other methods descriptions don't have an "s" after the verb -- should be "Approve an organization". There are actually several like this already, so maybe don't bother, or change them all...But at a quick glance it doesn't seem like we use the s form of the verb overall.